etherpad-debian/build-snowdrift.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+set -ex
+ETHERPAD_PLUGINS="ep_adminpads3 ep_chatdate ep_desktop_notifications ep_hide_referrer ep_small_list ep_comments_page ep_author_hover ep_webrtc"
```

Comment:
I get the following error:
```
npm ERR! code ENOVERSIONS
npm ERR! No valid versions available for ep_adminpads3

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/etherpad/.npm/_logs/2020-11-02T20_16_14_988Z-debug.log
```

Reply:
@ramereth try again with the latest commit, it looks like `ep_adminpads3` may have recently been moved to `ep_adminpads2`